solr/core/src/test/org/apache/solr/cloud/ForceLeaderTest.java

```diff
@@ -59,7 +59,7 @@ public static void beforeClassSetup() {
   @Test
   @Override
   @Ignore
-  public void test() throws Exception {}
+  public void test() {}
```

Comment:
no-op test, looks like. perhaps there are others elsewhere in the code base too? anyhow, outside the scope of current PR, just caught me eye.

Reply:
Good catch...